src/i18n/es.js

```diff
@@ -134,18 +134,25 @@ const messages_es = {
     message: `Mensaje`,
     new: `Nueva`,
     recaptcha_legal_MD: `Sitio protegido por reCAPTCHA. [Política de Privacidad](https://policies.google.com/privacy?hl=es) y [Condiciones del Servicio](https://policies.google.com/terms?hl=es) de Google.`,
+    save: `Guardar`,
     send: `Enviar`,
     show: `Mostrar`,
     synchronizing: `Syncronizando`,
     ui_library_image: `https://cdn.fromdoppler.com/doppler-ui-library/${dopplerUiLibraryVersion}/img/{imageUrl}`,
     unexpected_error: `¡Ouch! Ocurrio un error inesperado, intenta nuevamente`,
   },
   contact_policy: {
+    amount_description: 'No enviar mas de',
+    amount_unit: 'Emails',
+    interval_description: 'en un período de',
+    interval_unit: 'Días',
```

Comment:
I think that `Días` and `Emails` should be without capital letters. 
```suggestion
    interval_unit: 'días',
```

Reply:
This comes from the design defined in the figma.
https://www.figma.com/file/YjtGdoOSCoo4UTTtGxZowV/Pol%C3%ADtica-de-Contacto?node-id=27%3A5277
<img width="582" alt="Screen Shot 2021-06-22 at 14 24 47" src="https://user-images.githubusercontent.com/46003860/122979632-c67a6600-d365-11eb-8523-a48cc6b5468a.png">

Reply:
@mdirago is it ok like this? 🤔

Reply:
We tend to use capital letters only on Doppler entities to give them more presence, like Lists, Campaigns and so on. Here emails is referred to as a non Doppler entity, the same as days.

Reply:
Thanks for the clarification, that way the change makes more sense. Maybe @jmonacchi  can help us define what text we put.

Reply:
From https://makingsense.atlassian.net/wiki/spaces/DOP/pages/704249857/Pol+ticas+de+Contacto+V2
Text for inputs: `Enviar hasta XX Emails en un plazo de XX días`